concepts/match-basics/about.md

```diff
@@ -28,27 +28,20 @@ To handle multiple lines of code, the `match` arm of code should be wrapped in c
 
 ```rust
 fn specified_color(color: Color) -> felt252 {
+    let mut response: felt252 = '';
+
     match color {
         Color::Red => {
-            println!("The color is Red");
-            5
+            response = 'You passed in Red';
         },
         Color::Yellow => {
-            println!("The color is Yellow");
-            2
+            response = 'You passed in Yellow';
         },
         Color::Green => {
-            println!("The color is Green");
-            9
+            response = 'You passed in Green';
         },
     }
-}
-
-fn main() {
-let color = Color::Green;
-let value: felt252 = specified_color(color); // prints "The color is Green"
-
-println!("{}", value); // prints 9
+    response
 }
 ```
 
```

Comment:
Update the `main` function on [line 75](https://github.com/exercism/cairo/pull/104/files/2262bd41e55b1c0c3dcc1bebcc6d120dabeb0aa5..100e118c3ca6a31f5bf8c3d33a93201944d7b36c#diff-1374b5d5243b6ecabc1502e957b189589cc1233c0cd25bb57888cd8b2e0120d8R75) to the following:
```rust
fn main() {
    let color = Color::Red(ColorType::Dark);
    specified_color(color);
}
```
- removes redundant blank line
- removes the space char between `::` and `Dark`